sotodlib/mapmaking/noise_model.py

```diff
@@ -230,6 +230,75 @@ def from_bunch(data):
                 window=data.window, nwin=data.nwin, downweight=data.downweight,
                 bins=data.bins, D=data.D, V=data.V, iD=data.iD, iV=data.iV, s=data.s, ivar=data.ivar)
 
+class NmatWhite(Nmat):
+    def __init__(self, window=2, ivar=None, nwin=None):
+        """
+        This is a noise model for a bin map, with weights per det being the variance across time samples
+        
+        Parameters
+        ----------
+        window : int, optional
+            Size of the window in seconds to apply before taking the FFT and applying the model 
+            in harmonic space
+        ivar : numpy.ndarray or None, optional
+            Overwrite the inverse variance per detector
+        nwin: int or None, optional
+            Overwrite the window size in number of samples
+        
+        Returns
+        -------
+        noise_model : An Nmat object with the noise model
+
+        """
+        
+        self.ivar  = ivar
+        self.window     = window
+        self.nwin       = nwin
+        self.ready = ivar is not None
+    def build(self, tod, srate, **kwargs):
+        #ndet, nsamps = tod.shape
+        nwin  = utils.nint(self.window*srate)
+        ivar = 1.0/np.var(tod, 1)
```

Comment:
Yeah, the NmatWhite.build is generating weight from the variance of tod. At least people should aware that as I said above.

Reply:
Ok, this is much clearer now in the docstring.